### PR TITLE
Update PhpStorm to 2016.2

### PR DESCRIPTION
**Pull request by stevenjlho**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/phpstorm.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask 'phpstorm' do
-  version '2016.1.2'
+  version '2016.2'
   sha256 '8c2151b95903105d19d9a6d60aa4fd10ae7159c75680f1071dce8bd2fb900e4d'
```
Comment (fanquake) [inline Casks/phpstorm.rb]: You need to update the sha256

**Comment by vitorgalvao** [comment]
The cask has been on this version since may. You are way outdated. Run `brew update`.

**Comment by stevenjlho** [comment]
The newest version of cask is 2016.1.2. But The PhpStorm just release 2016.2 on July 13, 2016.

**Comment by vitorgalvao** [comment]
Oh, right, missed that.

Still, it seems like you did not run the commands in the boxes you ticked, since `sha256` was not ticked. Please submit again.

**Comment by vitorgalvao** [comment]
Done in https://github.com/caskroom/homebrew-cask/pull/22819.

**Comment by stevenjlho** [comment]
Thanks for your help.

**Pull request closed without merging**